frontend/aml_dashboard/src/index.js

```diff
@@ -0,0 +1,746 @@
+  })
+  .then(response => response.json())
+  .then(async json => {
+    console.log(json);
+    console.log('CREATED');
+
+    fiware_node_status.$value.set('<h1>Created !</h1> ');
+
+  });
+});
+
+// Post the data to the Context Broker
+post_data.$click.subscribe( async () => {
+
+  if (upload_data.$images.value == undefined)
+  {
+    throwError(new Error('No data has been uploaded'));
+  }
+  else
+  {
+    data_status.$value.set(' <h1>Sending...</h1> <br> <img src="https://i.gifer.com/Cad.gif">');
+    const image_data = Array.from(image.data);
+
+    fetch(url_context_broker + 'data', {
+      method: "POST", // *GET, POST, PUT, DELETE, etc.
+      mode: "cors", // no-cors, *cors, same-origin
+      cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+      credentials: "omit", // include, *same-origin, omit
+      headers: {
+        "Content-Type": "application/json",
+      },
+      redirect: "follow", // manual, *follow, error
+      referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+      body: JSON.stringify({
+        width : 6,
+        height : 6,
+        data : image_data
+      }) // body data type must match "Content-Type" header
+    })
+    .then(response => response.json())
+    .then(async json => {
+      console.log('SENDED');
+
+      data_status.$value.set('<h1>Sended !</h1> ');
+
+      solution_status.$value.set('<h1>Waiting for solution...</h1> ');
+
+      // Get the solution from the Context Broker
+      fetch(url_context_broker + 'solution', {
+        method: "GET", // *GET, POST, PUT, DELETE, etc.
+        mode: "cors", // no-cors, *cors, same-origin
+        cache: "no-cache", // *default, no-cache, reload, force-cache, only-if-cached
+        credentials: "omit", // include, *same-origin, omit
+        headers: {
+          "Content-Type": "application/json",
+        },
+        redirect: "follow", // manual, *follow, error
+        referrerPolicy: "no-referrer", // no-referrer, *no-referrer-when-downgrade, origin, origin-when-cross-origin, same-origin, strict-origin, strict-origin-when-cross-origin, unsafe-url
+      })
+      .then(response => response.json())
+      .then(async json => {
+        console.log(json);
+        console.log('SOLUTION RECEIVED');
+
+        var solution_attr = fiware_params.parameters['Attribute Solution'].value;
+        var solution = json[solution_attr];
+
+        var solution_string = JSON.stringify(solution, null, 4);
+        console.log(solution_string);
+
+        get_solution.$value.set(`<pre>${solution_string}</pre>`);
+
+        solution_status.$value.set('<h1>Solution received !</h1> ');
+
+      });
+    });
+  }
+});
+
+// Upload the data that will be sent to the Context Broker
+upload_data.$images.subscribe( async (img) => {
+
+  image = img;
+  data_status.$value.set(' <h1>Image uploaded</h1> ');
```

Comment:
Add a possible option for when there is no data received.


```suggestion
  data_status.$value.set(' <h1>Image could not be uploaded</h1> ');
```

Reply:
Same as the previous one.

Reply:
Done

Reply:
Addressed in #2